requirements.txt

```diff
@@ -1,6 +1,6 @@
 ordereddict==1.1
 python-dateutil==2.7
-matplotlib==1.3.1
-numpy==1.8.0
+matplotlib==2.2.2
```

Comment:
Surprised there aren't more breaking changes or deprecations to fix given the large difference in versions. If matplotlib generally has a stable API, should we perhaps pin to 2.2 instead of 2.2.2?

Reply:
I tried pinning to 2.2 and end up getting 2.2.0 installed